packages/components/RadioGroup/src/RadioGroup.tsx

```diff
@@ -76,6 +77,17 @@ export const RadioGroup = compose<IRadioGroupType>({
       return null;
     }
 
+    // Populate the buttonKeys array
+    if (children) {
+      /* eslint-disable @typescript-eslint/ban-ts-ignore */
+      // @ts-ignore - TODO, fix typing error
+      renderData.state.context.buttonKeys = React.Children.map(children, (child: React.ReactChild) => {
+        if (React.isValidElement(child)) {
+          return child.props.buttonKey;
```

Comment:
What if a user has something other than a RadioButton as a child to RadioGroup? What would the behavior be? Just thinking about edge cases

Reply:
Good call. I tested that scenario by adding a Button as a child of Radio Group in the test app (in between option "B" and "C"). It doesn't seem to effect desired result. When we map through each child and return `child.props.buttonKey`, it won't include any null or undefined values:

`map<T, C>(children: C | C[], fn: (child: C, index: number) => T):
            C extends null | undefined ? C : Array<Exclude<T, boolean | null | undefined>>;`
            
In this case, button doesn't have a prop called buttonKey so it doesn't include the value in the array buttonKeys. I assume it will do the same with any other child that's not a Radio Button (since it wouldn't have a prop called buttonKey either).

Reply:
And even if it did have a prop called buttonKey... I think that scenario would be really rare and kind of conflicting. Why would we have something other than RadioButton as a child of RadioGroup that also uses button keys 🤔 if we wanted to be super safe here, we could check the child to make sure it's a RadioButton before returning it's button key.

Reply:
Imo if it's only going to affect the accessibility props I don't think it's worth the effort to be really robust if it's unexpected that this case would happen. Until it's a legitimate scenario (or somehow it ends up being a common issue), I think we can let it be.

Reply:
Sounds good with me! Thanks for testing